BOOTSEQ.md

```diff
@@ -12,34 +12,53 @@ Entities optionally participating in the boot/reboot/shutdown processes:
 ## Asumptions
 
 - Resetting the xPU is assumed to cause PCIe errors that will crash the server.
+  - While this is a plausible scenario for a while, this will cause challenges for deploying an OS on the xPU and when running multi-xPU deployments. OPI should work with Linux Foundation to encourage host and OS developers to properly manage the response by the host and the xPU from a Reset
 - There are use cases for network boot only.
 - xPU boot can take some time (full linux distro)
 - There is communication between some of the entities from above
 - More...
 
 ## Bootup
 
-- Server is Powered On
-- xPU receives power and starts booting
-- Should Host OS wait for xPU to finish boot ?
-  - who tells Host OS to wait and how ?
+- Power events
+  - Server is Powered On
+  - xPU receives power and starts booting
+  - These events may not be synchronous. The xPU may be installed in a PICe slot providing persistent power. xPU power cycle may be a function of the host BMC (and potentially the xPU BMC) rather than the power state of the host
+- What is the boot order?
+  - Should Host OS wait for xPU to finish boot ?
+  - Should xPU OS wait for host to finish boot ?
+  - Should xPU and host be loosely coupled during boot phase?
+  - who tells Host OS/xPU OS to wait and how ?
   - who tells Host OS to to continue / start booting ?
   - is Host CPU halted ?
   - can we wait in UEFI/BIOS in case of network boot via xPU ?
 - Should host continue booting even if xPU failed to boot after timeout ?
+- What are the implications of one xPU failing boot in a multi-xPU deployment ?
 - More...
 
-## DPU reboot
+## xPU and host reboot
```

Comment:
I think we can leave this about xPU reboot only
Host reboot is handled in the next section `Graceful shutdown`